app/models/job_proxy_dispatcher.rb

```diff
@@ -93,6 +94,10 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def container_image_classes
+    ContainerImage.descendants.append(ContainerImage)
```

Comment:
hi @moolitayer, thanks... One thing that still confuses me is it feels like the vm scans are really VmOrTemplateScanJob things (hiding as the Job class) and the container image scans are really ContainerImageScanJob things (hiding as a list of descendent classes + self here)... I don't know why we didn't use single table inheritance on this table in the past but I'm wondering if we should do it now.  Perhaps fix this without a migration like you are here for any backports... but then, fix it right on master with STI.  What do you think?  cc @Fryguy 

The nice thing is then you can do ContainerImageScanJob.where... or VmOrTemplateScanJob.where as you would expect.  This might break the UI in many places where they're expecting Job classes by name, but it feels like we're doing it the hard way and fixing this might help maintenance in the future.